src/test/groovy/com/stormpath/tck/AbstractIT.groovy

```diff
@@ -134,6 +146,64 @@ abstract class AbstractIT {
         csrf = ret
     }
 
+    protected void setCSRFAndCookies(requestSpecification, contentType) {
+
+        if (Strings.hasText(csrf) && ContentType.JSON.equals(contentType)) {
+            requestSpecification.header("X-CSRF-TOKEN", csrf)
+        }
+
+        if (Strings.hasText(csrfKey) && Strings.hasText(csrf) && !ContentType.JSON.equals(contentType)) {
+            requestSpecification.formParam(csrfKey, csrf);
+        }
+
+        if (cookies != null) {
+            requestSpecification.cookies(cookies)
+        }
+    }
+
+    protected String getAccountHref(String email, String password) {
```

Comment:
No need to do this, see TestAccount.href:

https://github.com/stormpath/stormpath-framework-tck/blob/master/src/test/groovy/com/stormpath/tck/util/TestAccount.groovy#L23